docs/plugins/fluentbit/output/open_search.md

```diff
@@ -30,9 +30,9 @@ OpenSearch is the opensearch output plugin, allows to ingest your records into a
 | generateID | When enabled, generate _id for outgoing records. This prevents duplicate records when retrying OpenSearch. | *bool |
 | idKey | If set, _id will be the value of the key from incoming record and Generate_ID option is ignored. | string |
 | writeOperation | Operation to use to write in bulk requests. | string |
-| replaceDots | When enabled, replace field name dots with underscore, required by Elasticsearch 2.0-2.3. | *bool |
-| traceOutput | When enabled print the elasticsearch API calls to stdout (for diag only) | *bool |
-| traceError | When enabled print the elasticsearch API calls to stdout when elasticsearch returns an error | *bool |
+| replaceDots | When enabled, replace field name dots with underscore, required by Opensearch 2.0-2.3. | *bool |
+| traceOutput | When enabled print the Opensearch API calls to stdout (for diag only) | *bool |
+| traceError | When enabled print the Opensearch API calls to stdout when Opensearch returns an error | *bool |
```

Comment:
@Anghille you'll need to change it in the code below:
https://github.com/fluent/fluent-operator/blob/master/apis/fluentbit/v1alpha2/plugins/output/open_search_types.go#L85

and then run `make docs-update`
https://github.com/fluent/fluent-operator/blob/master/Makefile#L191C1-L191C12